DESCRIPTION

```diff
@@ -52,13 +52,16 @@ Depends:
 Imports:
     bayestestR (>= 0.13.0),
     effectsize (> 0.8.2),
-    insight (>= 0.18.8),
+    insight (>= 0.19.1),
     parameters (>= 0.20.0),
     performance (>= 0.9.2),
-    datawizard (>= 0.6.5),
+    datawizard (>= 0.7.0),
     stats,
     tools,
     utils
+Remotes:
+    easystats/effectsize,
```

Comment:
Why use remote for effectsize if required version is 0.8.2 and CRAN version is already 0.8.3?

Reply:
I actually just added the remotes for datawizard but that was not necessary, I don't know why there's also a remotes for effectsize, you can try to remove it